### PR TITLE
Add support for Apple Silicon based Macs

### PR DESCRIPTION
**Pull request by bubaumann**

### CODE REVIEW

File: maven/plugins/nodejs-maven-plugin/changes.xml [hunk 1]
```diff
@@ -22,7 +22,11 @@
 <document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/changes/1.0.0"
     xsi:schemaLocation="http://maven.apache.org/changes/1.0.0 http://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd">
   <body>
-
+    <release version="3.1.0">
```
Comment (stefanseifert) [inline maven/plugins/nodejs-maven-plugin/changes.xml:25]: i would rather make this version "3.0.2" and the action type to "update". it's not really adding a new feature, just extending some checks.

Comment (bubaumann) [reply maven/plugins/nodejs-maven-plugin/changes.xml]: I've changed it to 3.0.1, as this would be the next in line.

**Comment by tfiszerowicz** [comment]
Successfully tested on my machine.
MacBook Pro M1 (2020)
Big Sur 11.2.3 (20D91)
16GB

**Comment by stefanseifert** [comment]
thanks for the contribution!

**Pull request merged**